### ISSUE TITLE
Add URLs for Bitbucket

### ISSUE THREAD

**Issue by SchulteMarkus** [jenkinsci/embeddable-build-status-plugin#20]
Hi,

I cannot use "Markdown"-URLs from embeddable-build-status-plugin in Bitbucket, as 
[Bitbucket's image markdown syntax](https://bitbucket.org/tutorials/markdowndemo/overview#markdown-header-images) is somewhat different:

```
![Alt text](http://www.addictedtoibiza.com/wp-content/uploads/2012/12/example.png "Optional title")
```

Please add extra URLs "Bitbucket" at "Embeddable Build Status Icon" page.

**Comment by ramoantonioms** [comment]
![Alt text](clave.jpg)

**Comment by thomas-dee** [comment]
Resolved with PR #34
